src/creator/creator.service.ts

```diff
@@ -312,6 +317,12 @@ export class CreatorService {
         newFileKeys.push(logoKey);
         oldFileKeys.push(creator.logo);
       }
+      const creatorFiles: CreatorFiles = {
+        avatar: avatarKey,
+        banner: bannerKey,
+        logo: logoKey,
+      };
+      this.discordService.notifyCreatorFilesUpdate(creator.name, creatorFiles);
```

Comment:
you're creating an object which has a naming convention of an array. Why not just create an array?

`notifyCreatorFilesUpdate` can take in an array of files as a second parameter.
If we explicitly need to know what the type of a specific file from the array is (is some file a banner, avatar, or a logo), we can add a `type` property to the object.

e.g. take in an array of files, where each object contains data about the file + additional `type: "banner" | "avatar" | "logo"` property